devtools/it.xsemantics.workspace/feature.xml

```diff
@@ -1,20 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
We have copyright headers in feature.xml files in our repo.

PRO's for keeping the copyright headers in feature.xml:
1. during the IP review every file that does not have a copyright header at the top needs justification. And since the header does not do any harm in feature.xml, it is just simpler to have one,
2. keeping the standard copyright header for xml files also in feature.xml, means one special cases less,
3. also, strictly speaking, the copyright in the `<copyright>` tag does not necessarily cover the feature.xml files itself (yes, I know, it's kind of hairsplitting ;-).

So, for simplicity, I'd rather keep it (esp. due to 1. and 2.).